backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlTableFieldFactory.java

```diff
@@ -151,6 +151,15 @@ private GraphQLObjectType createTableObjectType(Table table) {
               GraphQLFieldDefinition.newFieldDefinition()
                   .name(id)
                   .type(GraphQLList.list(Scalars.GraphQLInt)));
+        case LONG:
+          tableBuilder.field(
+              GraphQLFieldDefinition.newFieldDefinition().name(id).type(Scalars.GraphQLBigInteger));
```

Comment:
actually, graphql doesn't officially know about LONG/BigInteger so officially this is deprecated feature